Pipeline/PackageBuildOutputs/PackageBuildOutputs.groovy

```diff
@@ -291,19 +334,29 @@ def parseInput(String[] cliArgs){
 
 	def props = new Properties()
 
+	// read properties file
+	if (opts.properties){
+		def propertiesFile = new File(opts.properties)
+		if (propertiesFile.exists()){
+			propertiesFile.withInputStream { props.load(it) }
+		}
+	}
+	
```

Comment:
Could you add an else block to read the default property file you shipped in your PR ? (to avoid the assertion later and add backward compatibility)

Reply:
default `packageBuildOutputs.properties` file is tried to be read, if the `--properties` option was not passed.